js/ui/components/clubhouse.js

```diff
@@ -507,6 +507,7 @@ var ClubhouseNotificationBanner = new Lang.Class({
                              { x: endX,
                                time: CLUBHOUSE_BANNER_ANIMATION_TIME,
                                transition: 'easeOutQuad',
+                               onUpdate: this._clipActor.bind(this),
```

Comment:
I wonder if instead of using `onUpdate()` you could tween the `clip` property of the actor itself. No worries if it's inconvenient.

Reply:
I don't know, I can't test this right now because I've not the external monitor here.